opengeode/AdaGenerator.py

```diff
@@ -434,9 +434,7 @@ def rec_detect_choice(expr: ogAST.Expression) -> bool:
             continue
         context_elems.append(f'{var_name.lower()} {type_name(var_type, False)}')
 
-    asn1_context = (f'\n{process_asn1}-Context ::='' SEQUENCE {\n'
-            + ",\n   ".join(line.replace("_", "-").replace("'", '"') for line in context_elems)
-            + "\n}\n")
+    asn1_context = (f'''\n{process_asn1}-Context ::='' SEQUENCE {'\n,\n   '.join(line.replace("_", "-").replace("'", '"') for line in context_elems)}\n\n''')
```

Comment:
I don't think this will work, did you test it ? If I recall correctly in f-strings you cannot do a '\n'.join(...)